telos-backend/demo.js

```diff
@@ -1,15 +1,39 @@
 const TodoTask = require('./src/models/todo_task');
 
+// Check if there are any TodoTask documents
```

Comment:
Could do "Creating demo data for TodoTask" instead of just saying what the code says.